core/client/app/controllers/application.js

```diff
@@ -25,23 +25,23 @@ export default Ember.Controller.extend({
     }),
 
     actions: {
-        topNotificationChange (count) {
+        topNotificationChange: function (count) {
             this.set('topNotificationCount', count);
         },
 
-        toggleAutoNav () {
+        toggleAutoNav: function () {
```

Comment:
I don't have any issue with the shortcut syntax but:

1. That goes against our existing style (no space between function name and parentheses), but the linter isn't yet smart enough to handle that particular bit of ES6 sugar.
2. We haven't had any actual discussion about what parts of ES6 we're going to use, what the style preferences will be and how that relates to the backend where we're still (realistically) stuck with ES5.

So for now I just reverted the few instances of these types of declarations.  I'm not asserting any sort of opinion aside from wanting to keep things consistent.